pkgs/tools/misc/present-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "present";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "terror";
+    repo = pname;
+    rev = version;
+    sha256 = "+kCHe84ikdCLd7j5YwP2j3xz+XTzzo/kLy+b9YUFDnI=";
+  };
+
+  cargoSha256 = "VKY/FQUrFWtLxKoK6LP6qPMqNN4absZvnAbH9mha1fI=";
+
+  doCheck = false;
```

Comment:
Please add a comment about why tests are disabled.

Reply:
When I looked into why tests were failing I realized it was just an unpatched shebang so fixed that and enabled tests.